.storybook/preview-head.html

```diff
@@ -0,0 +1,8 @@
+<link
```

Comment:
Added these for consistency with what's used in Layout. Now our fonts work in Storybook!

Reply:
Nice!